gui/test/projectfile/testprojectfile.cpp

```diff
@@ -20,7 +20,9 @@
 
 #include "addoninfo.h"
 #include "importproject.h"
+// cppcheck-suppress-begin uninitMemberVar
 #include "platform.h"
```

Comment:
I need to suppress all the warnings from an include. I would expect this to work but it doesn't as it was probably not intended to be used that way. I will file a ticket about it.